wemake_python_styleguide/visitors/ast/loops.py

```diff
@@ -338,6 +307,6 @@ def _check_implicit_items(self, node: ast.For) -> None:
 
     def _is_assigned_target(self, node: ast.Subscript) -> bool:
         parent = nodes.get_parent(node)
-        if not isinstance(parent, (*AssignNodes, ast.AugAssign)):
```

Comment:
Are you sure about this change? Why do you have to remove `ast.AugAssign`?

Reply:
Sorry, I don't remember changing that, I must've pulled an outdated version of the code. Fixed.